contracts/src/v0.8/functions/dev/1_0_0/HasRouter.sol

```diff
@@ -7,8 +7,6 @@ import {ITypeAndVersion} from "../../../shared/interfaces/ITypeAndVersion.sol";
 import {IOwnableRouter} from "./interfaces/IOwnableRouter.sol";
 
 abstract contract HasRouter is ITypeAndVersion, IConfigurable {
-  bytes32 internal s_configHash;
```

Comment:
We don't care about the config hash anymore?

Reply:
It didn't really do anything. It provided a safety check where you couldn't double apply a config, but that's not really a problem